### PR TITLE
Bump k8s 1.17 cnao to 0.32.0

### PR DESCRIPTION
**Pull request by qinqon**
It was using a bad multus version that does not support
--cni-version parameter at entrypoint.sh

Signed-off-by: Quique Llorente <ellorent@redhat.com>

### CODE REVIEW

**Comment by qinqon** [comment]
/hold 

Pending updating hash.

**Comment by qinqon** [comment]
/hold cancel

hash updated

File: cluster-up/cluster/images.sh [hunk 1]
```diff
@@ -6,7 +6,7 @@ declare -A IMAGES
 IMAGES[gocli]="gocli@sha256:220f55f6b1bcb3975d535948d335bd0e6b6297149a3eba1a4c14cad9ac80f80d"
 if [ -z $KUBEVIRTCI_PROVISION_CHECK ]; then
     IMAGES[k8s-fedora-1.17.0]="k8s-fedora-1.17.0@sha256:5fc78a20fae562ce78618fc25d0a15acd6de384b27adc3b6cd54f54f6c9d4fdf"
-    IMAGES[k8s-1.17]="k8s-1.17@sha256:54d04607e384c2bf1d1d837f21fe5106a31f1b2e09dc3d67c1bbf3c078dae930"
+    IMAGES[k8s-1.17]="k8s-1.17@sha256:1f4a7006f677ef2bd9def93c01dc879fbbffe3eea50d7c66210d787e7b009932"
```
Comment (maiqueb) [inline cluster-up/cluster/images.sh:9]: @qinqon where do I get this new sha256 from ?.. I'm missing that

**Pull request merged**